api/models/model_class_possibilities/Model_id_recursion.py

```diff
@@ -0,0 +1,60 @@
+# for more detail, see class Model in :
+# https://github.com/pallets/flask-sqlalchemy/blob/master/flask_sqlalchemy/model.py
+from typing import Any, List, Type
+
+
+class MyModel(object):
+    query_class = None  # flask_alchemy attribute
+    query = None  # flask_alchemy attribute
+
+    DONOTSEND_MODEL = {'_sa_instance_state'}
+    DONOTSEND = []
+    _jsonified = None
+
+    def __repr__(self) -> str:
+        return '<{}>'.format(self.__class__.__name__)
+
+    def to_dict_recursive(self) -> dict:
+        return self._to_dict_recursive(list_objects_id_passed_through=[id(self)])
+
+    def _to_dict_recursive(self, list_objects_id_passed_through: List[int]) -> dict:
```

Comment:
i like the typing here! I appreciate the thoughtfulness. 

Nit:
`list_objects_id_passed_through` is a little verbose. Maybe make it `obj_ids` or something like that?